tests/test_chat.py

```diff
@@ -183,6 +183,13 @@ def test(*args, **kwargs):
         monkeypatch.setattr('telegram.Bot.set_chat_administrator_custom_title', test)
         assert chat.set_administrator_custom_title(42, 'custom_title')
 
+    def test_unpin_all_chat_messages(self, monkeypatch, chat):
+        def test(*args, **kwargs):
```

Comment:
I recently took the habit of giving these internal functions a more speaking name :D maybe use `assertion` or `make_assertion`